src/skydb.ts

```diff
@@ -56,34 +56,24 @@ export type CustomGetJSONOptions = CustomGetEntryOptions &
   };
 
 export const DEFAULT_GET_JSON_OPTIONS = {
-  ...defaultBaseOptions,
-  ...defaultGetEntryOptions,
-  ...defaultDownloadOptions,
+  ...DEFAULT_BASE_OPTIONS,
+  ...DEFAULT_GET_ENTRY_OPTIONS,
+  ...DEFAULT_DOWNLOAD_OPTIONS,
   cachedDataLink: undefined,
 };
 
-/**
- * @deprecated please use DEFAULT_GET_JSON_OPTIONS.
- */
-export const defaultGetJSONOptions = DEFAULT_GET_JSON_OPTIONS;
-
 /**
  * Custom set JSON options.
  */
 export type CustomSetJSONOptions = CustomGetJSONOptions & CustomSetEntryOptions & CustomUploadOptions;
 
 export const DEFAULT_SET_JSON_OPTIONS = {
```

Comment:
It's probably nothing but this caught my eye:
```
export const DEFAULT_GET_JSON_OPTIONS = {
  ...DEFAULT_BASE_OPTIONS,
  ...DEFAULT_GET_ENTRY_OPTIONS,
  ...DEFAULT_DOWNLOAD_OPTIONS,
  cachedDataLink: undefined,
};
```
vs
```
export const DEFAULT_SET_JSON_OPTIONS = {
  ...DEFAULT_BASE_OPTIONS,
  ...DEFAULT_GET_JSON_OPTIONS,
  ...DEFAULT_SET_ENTRY_OPTIONS,
  ...DEFAULT_UPLOAD_OPTIONS,
};
```
that does not seem logical, shouldn't the `GET_JSON` include base, get entry, get json, download
and the `SET_JSON` include base, set entry, set json, upload?

You can just resolve this comment if that's not the case.

Reply:
It's that way because:
- getJSON gets an entry and downloads the file
- setJSON uploads a file, gets JSON for the current revision, and sets the entry
I'll add comments here for clarity.